TimberAPI/Dependency/Dependency.cs

```diff
@@ -7,17 +7,19 @@
 using TimberbornAPI.Common;
 using UnityEngine;
 
-namespace TimberbornAPI.Dependency
+namespace TimberbornAPI.DependencySystem
```

Comment:
I would prefer `DependencyInjection` or just `Dependencies`. But `DependencySystem` is definitely an improvement over `Dependency`.